app/components/faq/AboutChallenge.component.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Grid, List, Header, Icon } from 'semantic-ui-react';
+import './style.scss';
+
+const AboutChallenge = () => (
+  <React.Fragment>
+    <Grid id="about-challenge" stackable centered className='question-section' style={{ paddingTop: '2rem' }}>
+      <Grid.Column width={12} className='bold-box' style={{ paddingBottom: '3rem', paddingTop: '2rem' }}>
+        <Header as='h1' textAlign='center'>What's the challenge about?</Header>
+        <List>
```

Comment:
The content here isn't a list so it's [semantically wrong](https://www.w3schools.com/html/html5_semantic_elements.asp) to use <List/> and <List.item/>. Can you change this to <Container/> with <p/> or something similar?